### ISSUE TITLE
Makefile completion has missing require

### ISSUE THREAD

**Issue by accelbread** [JonWaltman/pcmpl-args.el#30]
After using `(require 'pcmpl-args)` as in the README, tab completing make gives me the following error:
```
Error in post-command-hook (corfu--post-command): (void-function pcmpl-gnu-make-rule-names)
```

Adding a `(require 'pcmpl-gnu)` fixes this for me.

Is there some setup step I'm missing, or is this a bug with the package?

**Comment by LemonBreezes** [comment]
No, I looked into this and yes, `(require 'pcmpl-gnu)` was missing so I added it to the code. Thank you for reporting it.
